reference/tools/tiup/playground.md

```diff
@@ -0,0 +1,110 @@
+      --pd int                   Specifies the number of PD instances in the cluster, 1 by default.
+      --pd.binpath string        Specifies the location of PD binary files, for development and debugging. You can ignore this flag.
+      --pd.config string         Specifies PD configuration files, for development and debugging. You can ignore this flag.
+      --tiflash int              Specifies the number of TiFlash instances in the cluster, 0 by default.
+      --tiflash.binpath string   Specifies the location of TiFlash binary files, for development and debugging. You can ignore this flag.
+      --tiflash.config string    Specifies TiFlash configuration files, for development and debugging. You can ignore this flag.
+```
+
+## Examples
+
+### Use the nightly version to start a TiDB cluster
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup playground nightly
+```
+
+In the command above, `nightly` is the version number of the cluster. Similarly, you can replace `nightly` with `v4.0.0-rc`, and the command is `tiup playground v4.0.0-rc`.
+
+### Start a cluster with monitor
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup playground nightly --monitor
+```
+
+This command starts Prometheus on port 9090 to display the time series data in the cluster.
+
+### Override PD's default configuration
+
+Copy the [PD configuration template](https://github.com/pingcap/pd/blob/master/conf/config.toml). If you copy the configuration file to the `~/config/pd.toml` path and make some changes according to your need, execute the following command to override PD's default configuration:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup playground --pd.config ~/config/pd.toml
+```
+
+### Replace the default binary files
+
+By default, when playground is started, each component is started using the binary files from the official image package. If you want to put a temporarily compiled local binary file into the cluster for testing, you can use the `--{comp}.binpath` flag for replacement. For example, execute the following command to replace the binary file of TiDB:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup playground --db.binpath /xx/tidb-server
+```
+
+### Start multiple component instances
+
+By default, only one instance is started for TiDB/TiKV/PD. To start multiple instances for each component, add the following flag:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup playground v3.0.10 --db 3 --pd 3 --kv 3
+```
+
+## Quickly connect to the started playground cluster
+
+TiUP provides the `client` component, which is used to automatically find and connect to a locally-started playground cluster. The usage is as follows:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup client
+```
+
+This command provides a list of playground clusters that the current machine has started on the console. Select the playground cluster to connected. After clicking <kbd>Enter</kbd>, a built-in MySQL client is opened to connect to TiDB.
```

Comment:
Needs update according to https://github.com/pingcap/docs-cn/pull/2993/files. @TomShawn